include/nbl/builtin/hlsl/workgroup/fft.hlsl

```diff
@@ -24,12 +25,37 @@ uint32_t3 gl_WorkGroupSize() {
 namespace workgroup
 {
 
+// ---------------------------------- Utils -----------------------------------------------
+
+template<typename SharedMemoryAccessor, typename Scalar>
+void exchangeValues(NBL_REF_ARG(complex_t<Scalar>) lo, NBL_REF_ARG(complex_t<Scalar>) hi, uint32_t threadID, uint32_t stride, NBL_REF_ARG(MemoryAdaptor<SharedMemoryAccessor>) sharedmemAdaptor)
```

Comment:
namespace/scope it better, right now its hanging around in `workgroup` namespace